clang/test/dpct/cmake_migation/case_014/expected.txt

```diff
@@ -0,0 +1,26 @@
+set(CPP_VER_98 98)
+set(CPP_VER_11 11)
+set(CPP_VER_14 14)
+set(CPP_VER_17 17)
+set(CPP_VER_20 20)
+set(CPP_VER_23 23)
+set(CPP_VER_26 26)
+
+# In the following cases check for changes after running the pattern-rewritter
+set (CMAKE_CXX_STANDARD 17)
+set (CMAKE_CXX_STANDARD 17)
+set (CMAKE_CXX_STANDARD 17)
+# Using magic number to be diverse
+set (CMAKE_CXX_STANDARD 17)
+
+# In the following cases check for NO changes after running the pattern-rewritter
+set (CMAKE_CXX_STANDARD CPP_VER_17)
+set (CMAKE_CXX_STANDARD CPP_VER_20)
+set (CMAKE_CXX_STANDARD CPP_VER_23)
+set (CMAKE_CXX_STANDARD CPP_VER_26)
```

Comment:
This four lines should be changed in current migration implementation.
It will make lit test failed.